src/client.js

```diff
@@ -1740,16 +1740,51 @@ class Client {
     async getSchema(schemaId, representation, internal) {
         var entity = await this._entityCache.get("xtk:schema", schemaId);
         if (!entity) {
-          entity = await this.getEntityIfMoreRecent("xtk:schema", schemaId, "xml", internal);
-          if (entity) {
-            const impls = DomUtil.getAttributeAsString(entity, "implements");
-            if (impls === "xtk:persist" && schemaId !== "xtk:session" && schemaId !== "xtk:persist") {
-                // Ensure xtk:persist is present by loading the xtk:session schema
-                await this.getSchema("xtk:session", "xml", true);
+              // special case of "temp:group:*" schemas for nms:group
+              // Schema "temp:group:*" is not cached because life cycle of this kind of schema is not the same as the others schemas
+              if (schemaId.startsWith("temp:group:")) {
+                  const parts = schemaId.split(":");
+                  var queryDef = {
+                    "schema": "nms:group",
```

Comment:
Use const

Reply:
I will use let because queryDef is assign below :
`queryDef = this._convertToRepresentation(queryDef, "SimpleJson", "xml");`